### DISCUSSION TITLE
Проблема после перехода на джумла 4

### DISCUSSION THREAD

**Discussion by FxBullMarket** [community/community#32153] [Other Feature Feedback, Questions, & Ideas]
Hello, I need help, after switching from 3.10.x to joomla 4 (version 4.2.1) for some reason, registered users can log in to the site, but new ones cannot register. I removed the old plugins and modules because there was no version 4. I have temporarily enabled the standard joomla component, but it gives a short error "Registration failed" (Registration failed:). And after the transition there are many problems, but now it is more important.
The topic is open on the official Jumla forum, but no one can find a solution. We checked the site folders and the database on the hosting, everything is in order there.

**Comment by queenofcorgis** [comment]
Hi there - can you elaborate more on how GitHub is affected and any other details related to GitHub please 😄
